### PR TITLE
Refactor UIColor category to use just one method for light/darken color

### PR DESCRIPTION
**Pull request by theocalmes**

### CODE REVIEW

File: BotKit/Categories/UIColor+AdjustColor.m [hunk 1]
```diff
@@ -10,43 +10,23 @@
...
     BOOL converted = [self getHue:&hue saturation:&saturation brightness:&brightness alpha:&alpha];
-
+    
     if (!converted)
         return nil;
-
-    CGFloat adjustmentAmount = amount;
-
+    
+    CGFloat adjustmentAmount = fabsf((amount));
```
Comment (hyperspacemark) [inline BotKit/Categories/UIColor+AdjustColor.m]: Just out of curiosity, what's the reasoning for the double parentheses?

File: BotKit/Categories/UIColor+AdjustColor.m [hunk 2]
```diff
@@ -10,43 +10,23 @@
...
+    
     if (adjustmentAmount < 0.0f) adjustmentAmount = 0.0f;
     if (adjustmentAmount > 1.0f) adjustmentAmount = 1.0f;
-
-    saturation += adjustmentAmount;
-    brightness -= adjustmentAmount;
-
+    
+    saturation += adjustmentAmount * ((amount<0) - (amount>0));
```
Comment (hyperspacemark) [inline BotKit/Categories/UIColor+AdjustColor.m]: Spaces between variables, values and operators.

```
((amount < 0) - (amount > 0));
```

**Comment by hyperspacemark** [comment]
Looks good. :thumbsup:

File: BotKit/Categories/UIColor+AdjustColor.m (hunk 1 above)
Comment (theocalmes) [inline BotKit/Categories/UIColor+AdjustColor.m]: thats a typo

**Pull request merged**